include/boost/url/authority_view.hpp

```diff
@@ -536,8 +536,7 @@ class BOOST_SYMBOL_VISIBLE
     template<
         class Allocator =
             std::allocator<char>>
-    string_value
-    userinfo(
+    const_string userinfo(
```

Comment:
I put the type on its own line

Reply:
OK. I'm on it. I'm going to miss clang-format on this PR. :laughing: